dev/CommonStyles/Deprecated_themeresources.xaml

```diff
@@ -179,6 +188,11 @@
             <StaticResource x:Key="ExpanderDisabledForeground" ResourceKey="TextFillColorDisabledBrush" />
             <StaticResource x:Key="ExpanderDisabledBorderBrush" ResourceKey="TextFillColorDisabledBrush" />
             <Thickness x:Key="ExpanderBorderThickness">2</Thickness>
+
+            <CornerRadius x:Key="ListViewItemCornerRadius">4</CornerRadius>
+            <CornerRadius x:Key="ListViewItemCheckBoxCornerRadius">4</CornerRadius>
```

Comment:
Previous resource value was 3, why is it 4 now?

Reply:
I changed it because I made the checkboxes use ControlCornerRadius. If you used these resources in other places you would keep the 3px radius, when the actual control would have 4px.

Reply:
But how would I get the value of 3 for this resource if we are now changing the value to 4? Not sure if this is truly deprecating the resource if we are also changing it's value.

Reply:
I meant to say that you would get 3 if I didn't change it.
By changing it to 4, if you used the resource for other controls inside Grid/ListViewItems, they would be adjusted alongside the checkboxes.

Reply:
And would it be the best way to deprecate the resource with also changing it's value? Not sure if we are breaking folks with this (which is something I as a dev wouldn't expect if something would be deprecated)

Reply:
Some things get a last update before they get deprecated, but it may not be the best practice. Now it depends on what people expect:
A) checkboxes inside ListView and other custom controls using that resource will have the same corner radius
B) custom controls using the resource will have a 3px corner radius

Reply:
Yes I guess it comes down what is the bigger priority here. I would side with keeping the resource (and option b), but that's my opinion.